deploy/3_deploy_Payments.js

```diff
@@ -0,0 +1,66 @@
+const hre = require("hardhat");
+
+module.exports = async () => {
+
+  const WETH = await hre.deployments.get("WETH");
+  const WMATIC = await hre.deployments.get("WMATIC");
+
+  const WETHAddress = WETH.address;
+  const WMATICAddress = WMATIC.address;
+
+  const ethUsdBytes32 = "0x4554482f55534400000000000000000000000000000000000000000000000000";
+  const maticUsdBytes32 = "0x4d415449432f5553440000000000000000000000000000000000000000000000";
+
+    const dapiServerAddress = "0x71Da7A936fCaEd1Ee364Df106B12deF6D1Bf1f14";
+
+    const PaymentsContract = await hre.deployments.deploy("Payments", {
+      args: [dapiServerAddress],
+      from: (await hre.getUnnamedAccounts())[0],
+      log: true,
+    });
+    console.log(`Deployed PaymentsContract at ${PaymentsContract.address}`);
+    console.log(`PaymentsContract contract deployed!`);
+    const paymentsContract = new hre.ethers.Contract(
+        PaymentsContract.address,
+        PaymentsContract.abi,
+      hre.ethers.provider.getSigner()
+    );
+
+  // Add allowed tokens
+  const addWETH = await paymentsContract.addAllowedTokens(WETHAddress);
+  await addWETH.wait();
+  console.log(`Wrapped ETH added to allowed tokens!`);
```

Comment:
this isn't how you do tests, you need to test if the state of the contract has changed. refer to this: https://hardhat.org/tutorial/testing-contracts and https://www.chainshot.com/article/how-to-write-unit-tests and maybe see other DAO contracts later to see the pattern we follow